server/auth.go

```diff
@@ -41,7 +41,7 @@ const (
 
 // Internal contains information about organization ID
 type Internal struct {
-	OrgID string `json:"org_id"`
+	OrgID uint64 `json:"org_id,string"`
```

Comment:
actually it is `int32`, limit set by Postgres

Reply:
basically positive subset of `int32` to be more precise, ugh

Reply:
but in types we have:
```Go
type OrgID uint64
```

Reply:
it is wrong, at least on storage side, and causes strange errors

Reply:
ie. might be fixed elsewhere